cves/kernel/CVE-2017-5548.yml

```diff
@@ -389,37 +412,39 @@ lessons:
     If you think of another lesson we covered in class that applies here, feel
     free to give it a small name and add one in the same format as these.
   defense_in_depth:
-    applies:
+    applies: False
     note:
   least_privilege:
-    applies:
+    applies: False
     note:
   frameworks_are_optional:
-    applies:
+    applies: False
     note:
   native_wrappers:
-    applies:
+    applies: False
     note:
   distrust_input:
-    applies:
+    applies: False
     note:
   security_by_obscurity:
-    applies:
+    applies: False
     note:
   serial_killer:
-    applies:
+    applies: False
     note:
   environment_variables:
-    applies:
-    note:
+    applies: True
+    note: | 
+      There seems to be added environmental variable that have been added in order to 
```

Comment:
This needs to be changed to either "an added environmental variable" or "added environmental variables" in order to make sense.  Not sure if this is trying to refer to one or multiple.  Consider changing the wording to remove the second added to something along the lines of "An environmental variable was added in order to fix the memory allocation problems."